doc/pages/FAQ.md

```diff
@@ -62,7 +62,35 @@ remove) are also run in a sandbox and guaranteed not to affect your system.
 
 ---
 
-#### 🐫  Why does ``opam init`` need to add stuff to my init scripts / why is ``eval $(opam env)`` needed ?
+#### 🐫  Why does opam require ``bwrap`` installed ?
+
+Since opam 2.0.0~rc2, opam uses `bwrap` in Linux to run package instructions in
+a sandbox. See [bubblewrap page](https://github.com/projectatomic/bubblewrap)
+for details.
+
+These sandboxing instruction are specified in the built-in configuration, that
+you can display with `opam init --show-default-opamrc`:
+
+```
+init-scripts: ["sandbox.sh" """ [...] """] {os = "linux"}
+
+wrap-build-commands: ["%{hooks}%/sandbox.sh" "build"] {os = "linux"}
+wrap-install-commands: ["%{hooks}%/sandbox.sh" "install"] {os = "linux"}
+wrap-remove-commands: ["%{hooks}%/sandbox.sh" "remove"] {os = "linux"}
+```
+
+You can manually disable package build sandboxing and remove bwrap from
+required dependencies (at your own risk). You can use the built-in
+configuration as a template to set up an `opamrc` file: run `opam init
+--show-default-opamrc >~/.opamrc`, edit it and retry your `opam init`.
+
+See also [wrap entry](Manual.html#configfield-wrap-build-commands) in the
+manual.
+
+---
+
+#### 🐫  Why does ``opam init`` need to add stuff to my init scripts / why is
+``eval $(opam env)`` needed ?
```

Comment:
Breaking the line here also breaks the formatting. I don't think Markdown has anyway of breaking a heading over several lines.